### ISSUE TITLE
Love Clonezilla but wish there was a nice easy to use GUI front end for it

### ISSUE THREAD

**Issue by phd21** [stevenshiau/clonezilla#45]
Dear Clonezilla developers,

First, I Love Clonezilla and Thank you for providing it for everyone to use. But, I do wish there was a nice easy to use GUI front-end for it like Norton Ghost an Acronis, etc... 

I have been using Clonezilla Live for years in Linux and it has always worked well for me. But, I also help out on the Linux Mint forums and I recommend Clonezilla to a lot of people. Unfortunately, a lot of new inexperienced users have trouble with its text interface.

Best regards,
Phil (phd21)

**Comment by stevenshiau** [comment]
Thanks for you suggestion. Yes, we knew that, but since we are so busy, too many projects involved... And only a few developers. Therefore for the time being, we still focus on the TUI development. For GUI, we will keep that in mind. 

Steven

**Comment by AE720** [comment]
Until that happens, I can at least suggest using a backup tool that boots into a linux environment that supports using clonezilla images here: https://rescuezilla.com/ I know it's not the answer you were looking for but it's better than nothing.

**Comment by stevenshiau** [comment]
FOSS is diverse, and Shasheen Ediriweera has a project which you might be interested:

https://github.com/rescuezilla/rescuezilla

@shasheene, thanks!

Steven

**Comment by shasheene** [comment]
Yep, Rescuezilla is an easy-to-use disk imaging application that's fully compatible with Clonezilla!

<a href="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/2.png"><img width=400 src="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/2.png" alt="Easy point and click interface anyone can use"></a> <a href="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/3.png"><img width=400 src="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/3.png" alt="Backups up only the used space"></a> <a href="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/6.png"><img width=400 src="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/6.png" alt="Many powerful tools available from the start button"></a> <a href="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/7.png"><img width=400 src="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/7.png"  alt="Provides browser access even if you can't log into your PC"></a> <a href="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/4.png"><img width=400 src="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/4.png" alt="Save to hard drive, network shared folder, or FTP server"></a> <a href="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/1.png"><img width=400 src="https://raw.githubusercontent.com/rescuezilla/rescuezilla.github.io/master/media/screenshots/1.png" alt="Select video mode, check CD integrity, or test RAM at boot"></a>

It's not yet 100% feature complete with Clonezilla yet, but it's still very useful to a lot of people!
